current-events/activity-current-events.md

```diff
@@ -0,0 +1,160 @@
+# Activity: Current Events
+
+## Goal
+
+Our goal is to create intentional space to discuss current events in technology.
+
+We want to become part of the conversation between technologists that evolves every day.
+
+Through discussion, we should aim to gain awareness of different topics and share what current events are interesting to us.
+
+Everyone will prepare a presentation of three or less slides to introduce a current event topic of your choice.
+
+## Preparation
+
+A slide deck of three or fewer slides should be prepared before activity day. Slide deck requirements are below.
+
+On the day of presentations, we should be prepared to discuss, present, and share in small groups.
```

Comment:
"activity day" sounds like it should be "Activity Day". soften with the article
"below" on its own sounds a little terse to me
changing the description from activity to presentations sounds like the presentation day might be a different day? stick with activity to make it clear this is all the same day
```suggestion
A slide deck of three or fewer slides should be prepared before the activity day. Slide deck requirements are listed below.

On the day of the activity, we should be prepared to discuss, present, and share in small groups.
```